core/benchmarks/KryoBenchmark-jdk17-results.txt

```diff
@@ -2,27 +2,27 @@
-basicTypes: Float with unsafe:true                     264            266           1          3.8         264.5       1.0X
-basicTypes: Double with unsafe:true                    269            272           5          3.7         268.6       1.0X
-Array: Int with unsafe:true                              3              3           0        365.9           2.7      99.2X
-Array: Long with unsafe:true                             5              5           0        214.8           4.7      58.3X
-Array: Float with unsafe:true                            3              3           0        375.7           2.7     101.9X
-Array: Double with unsafe:true                           5              5           0        210.6           4.7      57.1X
-Map of string->Double  with unsafe:true                 37             37           1         27.4          36.5       7.4X
-basicTypes: Int with unsafe:false                      286            287           1          3.5         285.7       0.9X
-basicTypes: Long with unsafe:false                     301            315          17          3.3         300.8       0.9X
-basicTypes: Float with unsafe:false                    279            294          17          3.6         278.6       1.0X
-basicTypes: Double with unsafe:false                   284            285           1          3.5         283.7       1.0X
-Array: Int with unsafe:false                            19             19           0         53.6          18.6      14.5X
-Array: Long with unsafe:false                           29             29           0         34.7          28.8       9.4X
-Array: Float with unsafe:false                           7              8           0        133.5           7.5      36.2X
-Array: Double with unsafe:false                         13             13           1         77.2          13.0      20.9X
-Map of string->Double  with unsafe:false                38             38           0         26.4          37.9       7.2X
+basicTypes: Int with unsafe:true                       264            271          15          3.8         264.5       1.0X
+basicTypes: Long with unsafe:true                      300            303           2          3.3         299.9       0.9X
+basicTypes: Float with unsafe:true                     288            290           1          3.5         288.0       0.9X
+basicTypes: Double with unsafe:true                    293            300           4          3.4         292.6       0.9X
+Array: Int with unsafe:true                              3              3           0        336.3           3.0      88.9X
+Array: Long with unsafe:true                             5              5           0        205.6           4.9      54.4X
+Array: Float with unsafe:true                            3              3           0        324.1           3.1      85.7X
+Array: Double with unsafe:true                           5              5           0        207.2           4.8      54.8X
```

Comment:
Seems obvious performance regression?

Reply:
Given that we change Java version and OS, this small change look okay to me~
```
- Array: Double with unsafe:true                           5              5           0        210.6           4.7      57.1X
+ Array: Double with unsafe:true                           5              5           0        207.2           4.8      54.8X
```